### DISCUSSION TITLE
Open in the external pdf reader not working for my old Foxit reader

### DISCUSSION THREAD

**Discussion by digdig** [sumatrapdfreader/sumatrapdf#4553] [General]
I am using sumatrapdf reader alongside with the old Foxit reader 4.3.  

For the new sumatrapdf reader release, I cannot open the external Foxit reader, although the menu has that option. However, in sumatrapdf 3.12, the functionality works fine. Therefore I always keep v3.12 and the latest release together. 

Is it possible for the developer add the code to open in external viewer as v3.12 to support my old Foxit reader? so that I don't need to  install v3.12 and only the newest version of sumatra?

My Foxit version:
![image](https://github.com/user-attachments/assets/94375b8d-43bd-4043-88ab-133dcb93ef44)

When I click open in Foxit reader with recent release of summtra, the error occurs ( but OK with v3.12 old release):
![image](https://github.com/user-attachments/assets/132b7325-b893-40d0-a8df-e8c26b234dab)

**Comment by GitHubRulesOK** [comment]
You can for now add your own if you know the path etc. so the hints are 

"path to foxitreader.exe" "%1" -n %p

should from a personal ExternalViewers entry add the option thus I have a copy remote from here thus I add
```
ExternalViewers [
 [
    CommandLine = "L:\!fromHP\Desktop\apps2\PDF\Foxit\FoxitReaderPortable.exe" /A nolock=1 "%1" -n %p
    Name = Remote Foxit Portable
    Filter = *.pdf
    Key = p
  ]
]
```
![image](https://github.com/user-attachments/assets/12bf785c-8674-4067-8ce5-f81e3e097ec7)

pressing p will open that page in that executable. 

![image](https://github.com/user-attachments/assets/febba551-76a8-4c34-a66a-edc0e0e80277)

**Reply by digdig** [reply]
Thanks for the reply, I managed to tweak with your code, and it works.

```
ExternalViewers [
 [
    CommandLine = "C:\Programs\Foxit Reader\Foxit Reader.exe" "%1" -n %p
    Name = Remote Foxit Portable
    Filter = *.pdf
    Key = p
  ]
]
```
